client/src/components/Transactions/Cell.jsx

```diff
@@ -0,0 +1,42 @@
+import React, {useEffect, useState} from "react";
+import styled from 'styled-components';
+
+
+const Cell = (props) =>{
+  const [rate, setRate] = useState(false);
+
+  return (
+
+    <Container>
+
+      <div>PHOTO: {props.item.pictures}</div>
+      <div>PRODUCT INFO: {props.item.title} {props.item.category} {props.item.description} </div>
+      <div>USER INFO: {props.user.first_name} {props.user.last_name} {props.user.user_name}</div>
+      {(props.user.charity_state && (props.group === 'donations' && props.status === 'donated')) &&
+        <button onClick={() => setRate(!rate)}>Rate User</button>}
+      {rate &&
+        <Modal className='modal'>
```

Comment:
I think we are trying to avoid global css with classes. I'm not sure what the purpose of className='modal' is. If it is a styling thing, try to move that to a styled-components fromat